docs/faq.rst

```diff
@@ -123,3 +123,22 @@ Yes, these techniques are complementary.
 Multiband fMRI leads to collecting multiple slices within a volume  simultaneously, while multi-echo
 fMRI is instead related to collecting multiple unique volumes.
 These techniques can be combined to reduce the TR in a multi-echo sequence.
+
+********************************************************************************
+[ME-fMRI] How do field maps and distortion correction work with multi-echo fMRI?
+********************************************************************************
+
+There are many different approaches to susceptibility distortion correction out there- far too many to discuss here.
+However, the good news is that distortion does not typically change across echoes in multi-echo fMRI.
+In most cases, the ``EffectiveEchoSpacing`` is the same across echoes,
```

Comment:
```suggestion
In most cases, the readout acquisition type and total readout time are the same for each echo,
```
"Echo spacing" is really confusing in this context because the space between each echo in a multi-echo sequence is also called the echo spacing. The real thing is that the readout is identical for all echoes (not just one parameter in the readout).